src/scene.h

```diff
@@ -72,7 +72,7 @@ class Scene {
 	 * executes the scene that is currently on the top of
 	 * the stack.
 	 */
-	virtual void MainFunction();
+	void MainFunction();
```

Comment:
made that change because it is on the stack when emscripten_sleep is called, so there shouldn't be multiple implementations of it -> simpler instrumentation